app/javascript/packages/document-capture/components/document-capture-troubleshooting-options.spec.tsx

```diff
@@ -149,34 +146,14 @@ describe('DocumentCaptureTroubleshootingOptions', () => {
         </FlowContext.Provider>
       );
 
-      it('has link to IPP flow', () => {
-        const { getByText, getByRole } = render(
-          <DocumentCaptureTroubleshootingOptions hasErrors />,
-          { wrapper },
-        );
-
-        expect(getByText('components.troubleshooting_options.new_feature')).to.exist();
-
-        const link = getByRole('link', { name: 'idv.troubleshooting.options.verify_in_person' });
-
-        expect(link).to.exist();
-      });
+      it('renders in-person call to action', () => {
+        const { getByRole } = render(<DocumentCaptureTroubleshootingOptions hasErrors />, {
+          wrapper,
+        });
 
-      it('logs an event when clicking the troubleshooting option', async () => {
```

Comment:
Yeah, since the logging is the responsibility of the new component, it felt appropriate to move the test cases there. This component is now only concerned that the component itself is actually rendered, reflected in the updated assertions. 

We also have Ruby-based integration specs which serve as a back-up as well:

https://github.com/18F/identity-idp/blob/2f37a9d9b2b21e9b34ae868fe1ab2adcc900fd07/spec/features/idv/analytics_spec.rb#L102